olive/utils.py

```diff
@@ -0,0 +1,10 @@
+def get_package_name_from_ep(execution_provider):
+    PROVIDER_PACKAGE_MAPPING = {
+        "CPUExecutionProvider": ("onnxruntime", "ort-nightly"),
+        "CUDAExecutionProvider": ("onnxruntime-gpu", "ort-nightly-gpu"),
+        "TensorrtExecutionProvider": ("onnxruntime-gpu", "ort-nightly-gpu"),
+        "RocmExecutionProvider": ("onnxruntime-gpu", "ort-nightly-gpu"),
```

Comment:
>ort-nightly-gpu [](http://example.com/codeflow?start=54&length=15)

seems we can replace onnxruntime with "ort-nightly" exception openvino?

Could we mapping to stable version and do the string replace in package except openvino?

Reply:
Even that is the pattern, I still treat them as different packages, like openvino is not following this, and I believe rocm package (to be released) will not follow this pattern as well.